lineage_TBX704.mk

```diff
@@ -41,7 +41,8 @@ PRODUCT_MANUFACTURER := Lenovo
 PRODUCT_GMS_CLIENTID_BASE := android-lenovo
 
 PRODUCT_BUILD_PROP_OVERRIDES += \
-    PRIVATE_BUILD_DESC="hq_msm8953_64-user 7.1.1 NMF26F 1581 release-keys" \
+    ro.build.version.security_patch=2018-10-01 \
```

Comment:
Hi @egodamager 

sorry again for needing that long to review this. Line 44 leads to build errors (`ro.build.version.security_patch=2018-10-01: command not found`). Without that line I can start the build.

The vendor patch is set [here](https://github.com/lukaspieper/android_device_lenovo_tb_x704f/blob/3889a31d9456a4bea4358f30a98511041b9cd80a/BoardConfig.mk#L209).